File: PlantsAPI/Models/ReplyDto.cs
namespace PlantsAPI.Models
{
    public class ReplyDto
    {
        public Guid Id { get; set; }
        public string Content { get; set; }
        public DateTime DateOfCreation { get; set; }

        public string Username { get; set; }
        public byte[] ProfileImage { get; set; }

        public Guid PostId { get; set; }

        public Guid UserId { get; set; }

    }
}

File: PlantsWeb/src/app/models/ReplyDto.ts
import { SafeUrl } from "@angular/platform-browser";

export class ReplyDto {
    constructor(
        public id : string,
        public content : string,
        public dateOfCreation : Date,
        public username : string,
        public profileImage : SafeUrl | string | null,
        public postId : string,
        public userId : string,
    ){}

}